lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -46,7 +46,17 @@ class GoogleCloudOutput < BufferedOutput
     CLOUDFUNCTIONS_SERVICE = 'cloudfunctions.googleapis.com'
```

Comment:
I haven't released any gem from this branch. `0.5.5.pre.1` was released from master. And we are planning to release `0.5.5` soon. So I will update this to `0.5.6.v2.alpha.1`.